### DISCUSSION TITLE
Can I resize the width of the pager?

### DISCUSSION THREAD

**Discussion by ziontee113** [jesseduffield/lazygit#2430] [Q&A] [answered]
Hi, can I somehow increase the size of the git diff pager? Thank you very much!

![Screenshot from 2023-02-10 14-48-04](https://user-images.githubusercontent.com/102876811/218033069-f112b0a3-bb0d-40cf-848b-40ed8b3a14a8.png)

**Comment by stefanhaller** [comment, marked as answer]
Yes, there's a `gui.sidePanelWidth` option that lets you set the ratio of side panel to main view in percent (or rather, as a factor from 0 to 1). The default is 0.333.

See https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md for everything that you can configure in lazygit.

**Reply by ziontee113** [reply]
Thank you very much :pray: , I'll try to find and read the docs better next time.

**Comment by texastoland** [comment]
Is there a way to expand it temporarily so side-by-side diff is usable on a small screen? Don't want to open an issue because I'm sure there's a good reason if not!

**Reply by texastoland** [reply]
I saw `enlargedSideViewLocation: top` in your video and that resolved it 🥳

**Reply by raminudelman** [reply]
The keyboard shortcut `Ctrl` + `Shift` + `=` works for me to cycle between "default" view, "half screen" view and "full screen" view.
